src/Microsoft.TemplateEngine.Core/ConditionalTokens.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.TemplateEngine.Core
+{
+    public class ConditionalTokens
+    {
+        public IList<string> IfTokens = new List<string>();
```

Comment:
Ah yeah, we had started talking about this yesterday. Done.

From: Mike Lorbetske [mailto:notifications@github.com]
Sent: Tuesday, August 23, 2016 11:15 AM
To: seancpeters/microsoft-templateengine <microsoft-templateengine@noreply.github.com>
Cc: Sean Peters <sepeters@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [seancpeters/microsoft-templateengine] Refactor (#1)


In src/Microsoft.TemplateEngine.Core/ConditionalTokens.cs<https://github.com/seancpeters/microsoft-templateengine/pull/1#discussion_r75921684>:

> @@ -0,0 +1,19 @@

> +﻿using System;

> +using System.Collections.Generic;

> +using System.Linq;

> +using System.Threading.Tasks;

> +

> +namespace Microsoft.TemplateEngine.Core

> +{

> +    public class ConditionalTokens

> +    {

> +        public IList<string> IfTokens = new List<string>();

private static readonly IReadOnlyList NoTokens = new string[0];

public ConditionalTokens()
{
IfTokens = NoTokens;
ElseTokens = NoTokens;
...
}

public IReadOnlyList IfTokens { get; set; }
...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/seancpeters/microsoft-templateengine/pull/1/files/a425b2f9e55c8aa446463bd90060cb6bc99db3d0#r75921684>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AT8a8WJNJpC4iNOay2Y10185ikV0kvvCks5qizi8gaJpZM4JqaeQ>.